Source/String.swift

```diff
@@ -68,7 +68,7 @@ extension String {
     }
 
     public func trimLeft(_ characterSet: CharacterSet) -> String {
-        var start = characters.count
+        var start = 0
```

Comment:
Really? I thought of it more of where do we start? And since it's a trimLeft, we seek from the left, right? I.e. if there are no characters, the contents of the for cycle won't execute at all, so you want to end up with the full string, not with an empty string. 

The way I think of it is "what would happen if I take out the for cycle altogether?". I think the result should be, "we return the full string". Not, "we return an empty string".

Or am I missing something?